docs/variants/msi_z690/cpu-hcl.md

```diff
@@ -2,41 +2,158 @@
+      Profile data:
+        - Type/speed: for example DDR4-2400 means DDR4 module clocked at max 2400MHz
+        for given profile
+        - Timings: for example CL17-17-17 means CAS Latency 17, tRCD 17, tRP 17
+        (numbers expressed in clock cycles) for given memory profile
+        - Voltage: memory voltage in Volts for given memory profile
+
+    > NOTE: some XMP profiles may have lower speeds than other ones, but also have
+    > smaller CAS latency. Also the memory frequency configured in the memory
+    > controller and reported by DMI/SMBIOS may be slightly lower than advertised
+    > in the module specifications. This can vary based on the board design, CPU
+    > and many other factors.
+
+    ### HCL list - MSI PRO Z690-A DDR4
+
+    | Manufacturer | Part Number | Size | Type | Speed | Configuration 1/2/4 | Dasharo version | Source |
+    |:------------:|:-----------:|:----:|:----:|:-----:|:-------------------:|:---------------:|:------:|
+    | Crucial Technology | CT8G4DFS8266.C8FJ | 8192 MB | DDR4 SDRAM | 2666 MT/s (PC4-21300) | &#10004/-/- | v1.1.0 | Dasharo HCL report |
+    | Kingston | KF3600C17D4/8GX      | 8192 MB | DDR4 SDRAM | 2400 MT/s (PC4-19200) | &#10004/-/&#10004 | v1.0.0 | Dasharo HCL report |
+    | Kingston | KF3600C17D4/8GX      | 8192 MB | DDR4 SDRAM | 2400 MT/s (PC4-19200) | -/-/&#10004 | v1.1.0 | Dasharo HCL report |
+    | Patriot Memory | 4400 C19 Series | 8192 MB | DDR4 SDRAM | 2133 MT/s (PC4-17000) | &#10004/-/- | v1.0.0 | Dasharo HCL report |
+    | Thermaltake Technology Co Ltd | RG26D408GX2-3600C18A | 8192 MB | DDR4 SDRAM | 2666 MT/s (PC4-21300) | &#10004/-/- | v1.1.0 | Dasharo HCL report |
+
+    1) Tested in 2x2 mixed configuration
+
+### HCL list - MSI PRO Z690-A DDR5
+
+| DIMM vendor | Part Number | Size | SPD profile | Configuration 1/2/4 |
+|:-----------:|:-----------:|:----:|:-----------:|:-------------------:|
+| Kingston    | [KF556C40BB/8][6] |8GB | 5600MT/s 40-40-40 1.25V | &#10004;/&#10004;/&#10004; |
+| Crucial     | [CT8G48C40U5.M4A1/8][7] |8GB | 4800MT/s 40-39-39 1.1V | &#10004;/&#10004;/&#10004; |
+| ADATA       | [AD5U48008G-S][9] |8GB | 4800MT/s 40-40-40 1.1V | &#10004;/&#10004;/&#10004; |
+
+[1]: https://www.kingston.com/dataSheets/KF436C17BBK4_32.pdf
+[2]: https://www.corsair.com/eu/en/Categories/Products/Memory/VENGEANCE-LPX/p/CMK16GX4M2B3200C16
+[3]: https://www.kingston.com/dataSheets/KF432C16BB1_16.pdf
+[4]: https://www.pny.com/anarchy-x-ddr4-red?sku=MD16GK2D4320016AXR
+[5]: https://www.kingston.com/datasheets/KF436C18BBK2_64.pdf
+[6]: https://www.kingston.com/memory/gaming/kingston-fury-beast-ddr5-memory
+[7]: https://www.crucial.com/memory/ddr5/ct2k16g48c40u5
+[8]: https://www.kingston.com/datasheets/HX426C16FRK4_64.pdf
+[9]: https://www.adata.com/us/consumer/dram-module-ddr5-4800-u-dimm
+[10]: https://www.amazon.com/OLOy-Memory-288-Pin-Desktop-MD4U323216DJDA/dp/B0836NZHWR
+
+=== "GPU HCL"
+
+    GPU Hardware Compatibility List presents the GPUs tested and verified
```

Comment:
`tested and verified` may mean a lot. In the DDR section we explain a bit what do we mean by that.

Reply:
@macpijan Accurate, and in the context of GPU, it is not easy to claim about `tested and verified`. What do you suggest? Please note those are not my words. Those are words contributed by @sulewskiprzemyslaw approved and verified by us: https://github.com/Dasharo/docs/pull/208/files